src/main/java/at/hannibal2/skyhanni/features/garden/pests/SprayFeatures.kt

```diff
@@ -57,5 +63,13 @@ class SprayFeatures {
         config.position.renderString(display, posLabel = "Pest Spray Selector")
     }
 
+    @SubscribeEvent
+    fun onWorldRender(event: LorenzRenderWorldEvent) {
+        if (!config.drawPlotsBorderWhenInHands) return
+        if (InventoryUtils.getItemInHand()?.getInternalName()?.asString() != "SPRAYONATOR") return
```

Comment:
You can use `InventoryUtils.itemInHandId` here.

Reply:
I forgot, my bad